src/DataType/Reference/DocumentReference.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Tiime\FacturX\DataType\Reference;
+
+class DocumentReference
```

Comment:
```suggestion
abstract class DocumentReference
```
Sauf si y'a des références vers des documents dont on ne connais pas le type